Rocket.Chat/Controllers/Chat/ChatControllerMessageActions.swift

```diff
@@ -77,6 +79,12 @@ extension ChatViewController {
             self?.reply(to: message)
         }))
 
+        if AuthManager.isAuthenticated()?.canDeleteMessage(message) == .allowed {
+            alert.addAction(UIAlertAction(title: localized("chat.message.actions.delete"), style: .destructive, handler: { _ in
+                API.current()?.client(MessagesClient.self).deleteMessage(message, asUser: false)
```

Comment:
Why did you use `asUser = false` here? I think we should always use true 🤔. What am I missing? 🙈

Reply:
asUser = true doesn't work very well 🤷‍♂️ and asUser = false didn't pose any problems/differences.

Reply:
Ahhhh, I think I see what's going on here:

https://github.com/RocketChat/Rocket.Chat/blob/5c733384f2cc62437edaadf2df250b72c75aed54/packages/rocketchat-api/server/v1/chat.js#L20-L26

So, basically, you use that in case you want to delete with `userA` but the action looks like `userB` has done it? 🤔 

Anyway, if it works... just make sure to test with all kind of users (admins & normal users, basically).